docs/snippets/examples/DefinitionExample.java

```diff
@@ -18,6 +20,6 @@ public static void example(Stork stork) {
         // Another service using the random selection strategy, instead of round-robin
         stork.defineIfAbsent("my-second-service",
                 ServiceDefinition.of(new StaticConfiguration().withAddressList(example),
-                        new RandomConfiguration()));
+                        new RandomConfiguration(), new StaticRegistrarConfiguration()));
```

Comment:
why do we need a registar here.

Having a version of the method without would avoid a large breaking change.

Reply:
We don't. I will modify to have a method with 2 arguments (SD+LB)